diesel_cli/src/cli.rs

```diff
@@ -222,3 +252,19 @@ fn migration_dir_arg<'a, 'b>() -> Arg<'a, 'b> {
         .takes_value(true)
         .global(true)
 }
+
+fn migration_revert_number_validator(val: String) -> Result<(), String> {
```

Comment:
You don't need this whole function when you add the `all` argument. And you don't need the docs either since CLI help will show it.

Reply:
I added a `validators` module. For now it's used in `REVERT_NUMBER` to check if the value of the argument is a positive number.